client/src/components/RecipeList/index.js

```diff
@@ -20,11 +20,11 @@ const RecipeList = ({ recipes, recipeTitle }) => {
                             className = 'text-light'
                             >
                                 {recipe.username}
-                        </Link>{"' "}
+                        </Link>{" "}
```

Comment:
can you put back the 's so that when it renders, it looks like [username]'s recipe on ...